ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/TimeModel.scala

```diff
@@ -53,11 +53,25 @@ object TimeModel {
       maxSkew = Duration.ofSeconds(30L),
     ).get
 
+  val resolution: Duration = Duration.ofNanos(1000L)
```

Comment:
This is a constant, isn't it?
```suggestion
  val Resolution: Duration = Duration.of(1L, MICROS)
```

Reply:
Yes, but for consistency I'd also have to rename the preexisting one. I'll have a look.

Reply:
What do you mean by 'preexisting one'?